File: warden_web/public/app_js/model/test_case_run_info.js
Ext.define('WardenWeb.model.test_case_run_info', {
  extend: 'Ext.data.Model',
  fields: [ 'id', 'start_at', 'end_at', 'status', 'test_case_log', 'external_data',
    'number_of_steps', 'test_case',
    { name: 'feature_name', type: 'string', mapping: 'test_case.feature_name'},
    { name: 'name', mapping: 'test_case.name'},
    { name: 'feature_file_path', mapping: 'test_case.feature_file_path'},
    { name: 'tc_id', mapping: 'test_case.tc_id'},
    { name: 'project_name', mapping: 'test_case.warden_project.name'}
  ],

  belongs_to: 'test_case',
  belongs_to: 'test_run_job'
});

File: warden_web/app/models/test_case_run_info.rb
class TestCaseRunInfo < ActiveRecord::Base
  attr_accessible :end_at, :external_data, :number_of_steps,
    :start_at, :status, :tags, :test_case_log,
    :test_case, :test_run_job, :test_run_history,
    :screen_capture_links #relational links
  belongs_to :test_run_job
  belongs_to :test_case
  belongs_to :test_run_history

end
